pipx/main.py

```diff
@@ -722,6 +741,16 @@ def get_command_parser():
         help="The Python binary to associate the CLI binary with. Must be v3.3+.",
     )
 
+    p = subparsers.add_parser(
+        "inject", help="Install a package into an existing virtualenv"
+    )
+    p.add_argument(
+        "package", help="Name of the existing pipx-managed virtualenv to inject into"
+    )
+    p.add_argument("dependency", help="the package to inject into the virtualenv")
+    p.add_argument("--spec", help=SPEC_HELP)
```

Comment:
Spec is not needed since the package name is already provided. The dependency is already what pipx considers a spec, so this line should be removed and the other code that tries to use it should be removed too.